test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs

```diff
@@ -100,6 +101,41 @@ public void Defaults()
             Assert.Equal(ConfigurationManager<OpenIdConnectConfiguration>.MinimumRefreshInterval, new TimeSpan(0, 0, 0, 1));
         }
 
+        [Fact]
+        public void FetchMetadataFailureTest()
+        {
+            var context = new CompareContext($"{this}.FetchMetadataFailureTest");
+
+            var documentRetriever = new HttpDocumentRetriever(HttpResponseMessageUtils.SetupHttpClientThatReturns("OpenIdConnectMetadata.json", HttpStatusCode.NotFound));
+            var configManager = new ConfigurationManager<OpenIdConnectConfiguration>("OpenIdConnectMetadata.json", new OpenIdConnectConfigurationRetriever(), documentRetriever);
+
+            //First time to fetch metadata
```

Comment:
nit: Generally the style is to add a space after the // for comments. (one other occurance below as well)